app/helpers/select_helper.rb

```diff
@@ -9,6 +9,22 @@ def array_to_select_collection(array, transform_func = :humanize)
     end
   end
 
+  def user_roles_select_collection
+    array_to_select_collection(current_user_roles_to_select, :titleize)
+  end
+
+  # returns roles which current user can set when creating/editing other users
+  def current_user_roles_to_select
```

Comment:
When editing a user profile and the current_user has one of the editor roles the dropdown is showing "Yes" and "No" as the options which are not valid options. I guess we can make the role not editable in this case. This will only happen if they are editing their own account.

Reply:
Do we want this (disable role input when editing self) even for super admins?

Reply:
fixed in 25e7562 - when editing self, role input is disabled

Reply:
Meaning that only other super user can edit someone's role? We can call that a security feature :)